example/build_with_old_bazel/.bazelrc

```diff
@@ -0,0 +1,16 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
Do we need to delete all the files under `build_with_old_bazel`?

Reply:
It's to prove we can build brpc with old version of bazel. It's under example folder. We can delete it if we want.